### DISCUSSION TITLE
Node to combine images

### DISCUSSION THREAD

**Discussion by Hboni** [Hboni/francis#17] [Ideas]
I was thinking about adding processes that allow combining at least two images in the first version.
With the possibility to add, subtract, mask images ...

It needs to be able to have two inputs on the same node, with the possibility to choose which function to do (or create a node for each function). It could be great!

What do you think about that @jeaunrg? Do you think that it can be feasible? For a first version?

**Comment by jeaunrg** [comment]
Yes the code was waiting for that improvement! I just need to update a few things. 
I dont know if youve noticed but you can already select multiple nodes by holding ctrl, then adding a child node will create multiple links..

**Reply by Hboni** [reply]
No, I did not notice that it was possible, that sounds great! So if you agree, you can create a new branch for that, to develop this new feature of the new node to add, subtract, combine images?

**Reply by jeaunrg** [reply]
What do you mean when you say 'combine' images ? 
Should I add multiply and mask nodes (which are almost the same)?

**Reply by Hboni** [reply]
Yeah I most though about multiplication of images 😉
